### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: __unported__/bom_stock/__openerp__.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
     ,
     'author': 'Camptocamp',
     'website': 'http://www.camptocamp.com',
+    'license': 'AGPL-3',
```
Comment (yvaucher) [inline __unported__/bom_stock/__openerp__.py]: GPL-3 or later

**Comment by rvalyi** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

**Pull request merged**